modules/perception/base/point_cloud.h

```diff
@@ -246,7 +252,7 @@ class PointCloud {
   virtual bool CheckConsistency() const { return true; }
 
  protected:
-  std::vector<PointT> points_;
+  EigenVector<PointT> points_;
```

Comment:
Is it necessary to use EigenVector here?

Reply:
I copied this from pcl point_cloud.h and I have not try this one by one, pcl make aligned_allocator in every PointT vector.

Reply:
Rollback to the original version and test if it can still work.